src/stylesheets/components/_search-result.scss

```diff
@@ -2,3 +2,7 @@
   border-bottom: 1px solid $color-black;
   padding-bottom: 1em;
 }
+
+.usa-label {
```

Comment:
Wondering if we can try a different selector here:

1. Remove `.usa-grid` from the search result main declaration.
2. Move `usa-label` to a sub-definition of `usa-search-result`.

Wanting to make sure if we ever do something like:

```
<p>Hello, how are you <span class="usa-label">Anurag</span> and <span class="usa-label">Kapil</span>.</p>
```

Does not result in a right margin to the highlighted text. We are considering ways to highlight the relevant keywords in the search results and considering using the `usa-label` concept to help that along.